.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: release
+
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
```

Comment:
:+1:

Reply:
I never have liked to the little `v` in the tags but looking at releases that ship has sailed.